### ISSUE TITLE
Set screen resolution for application

### ISSUE THREAD

**Issue by dwenger01** [giantotter/giantotter_public#344]
A solution to this was introduced for Lemelson, but a more robust version needs to be created on the Master branch to properly support prefab cameras that will be used for spectatating (for example).

Solution:
In Lemelson, a script was added to the MainCamera_prefab object that 1) allowed the user to define Desktop and Android screen resolutions in a script class that would then set the proper screen resolution at camera creation time.

Extension:
The same solution can be used on the Master branch; however, at least one additional camera prefab object should be created that does not set the screen resolution on startup.  This is because spectating will use additional cameras that will be created from prefab objects, and we don't want to set/reset the screen resolution whenever a spectating camera is created.

**Comment by grahamboree** [comment]
Hey guys,
  I just found a really neat feature in uGUI that seems like it completely solves this problem.  The root problem we're solving is that the UI was too small on higher-resolution devices.  Turns out that there's a component called "CanvasScaler" which scales the UI proportional to the screen resolution.  This allows us to keep things relatively the same size on screen.  So, if the screen resolution is super huge, it'll scale up the UI to be proportional to it.  Conversely it can proportionately scale down the UI to fit a smaller resolution without having issues like the chat UI was covering the interaction position.
